cache/cache.go

```diff
@@ -0,0 +1,129 @@
+package cache
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/caarlos0/env"
+	"github.com/gomodule/redigo/redis"
+	"github.com/pkg/errors"
+)
+
+type config struct {
+	Server      string        `env:"CACHE_SERVER" envDefault:"127.0.0.1:6379"`
```

Comment:
Could you rename the env variables here from `CACHE_` to `REDIS_` ? It's easier for a reader to understand which cache server is used